quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/storage/database/jpa/QuarkusJpaConnectionProviderFactory.java

```diff
@@ -25,6 +25,8 @@
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.nio.file.Files;
+import java.nio.file.LinkOption;
```

Comment:
don't hate me @pedroigor , but this is an unused import 😢

Reply:
You missed a few others :) Also updated the test herein changed.